sdk/cosmos/azure-cosmos/azure/cosmos/aio/_cosmos_client.py

```diff
@@ -197,6 +198,7 @@ async def create_database(  # pylint: disable=redefined-builtin
 
         :param str id: ID (name) of the database to create.
         :keyword int offer_throughput: The provisioned throughput for this offer.
+        :paramtype offer_throughput: int or ~azure.cosmos.ThroughputProperties.
```

Comment:
this is what it should look like, the way the other keywords work when they have complex typings (look at match_condition or response_hook for instance). Please fix this in all other instances that need it.
```suggestion
        :keyword offer_throughput: The provisioned throughput for this offer.
        :paramtype offer_throughput: int or ~azure.cosmos.ThroughputProperties.
```